### PR TITLE
Remove routing context from Consumer Creation

### PR DESCRIPTION
**Pull request by ShubhamRwt**
This PR removes the use of `RoutingContext` from consumer creation step.

### CODE REVIEW

File: src/main/java/io/strimzi/kafka/bridge/quarkus/RestBridge.java [hunk 1]
```diff
@@ -201,11 +203,13 @@ public CompletionStage<Response> getOffsets(@PathParam("topicname") String topic
     @POST
     @Consumes(BridgeContentType.KAFKA_JSON)
     @Produces(BridgeContentType.KAFKA_JSON)
-    public CompletionStage<Response> createConsumer(@Context RoutingContext routingContext, @PathParam("groupid") String groupId, byte[] body) {
+    public CompletionStage<Response> createConsumer(@Context UriInfo uri,
+                                                    @Context HttpHeaders httpHeaders,
+                                                    @Context RoutingContext routingContext, @PathParam("groupid") String groupId, byte[] body) {
```
Comment (ppatierno) [inline src/main/java/io/strimzi/kafka/bridge/quarkus/RestBridge.java]: RoutingContext parameter can be removed, or?

**Review by ppatierno — COMMENTED** [review]
Have you tested it somehow even manually or debugging?

**Comment by ShubhamRwt** [comment]
@ppatierno Yes I did tested it by using headers like `x-forwared-proto`,` x-forwared-host` etc but yeah it would be great if we can sync since this is my first time using these headers

**Review by ppatierno — APPROVED** [review]
After testing together with an nginx instance in front of the bridge, this LGTM, thanks!

**Pull request merged**